utils/field.h

```diff
@@ -46,11 +46,26 @@ struct display_field {
 	struct list_head list;
 	enum display_field_id id;
 	const char *name;
+	const char *alias;
 	const char *header;
-	int length;
 	bool used;
+	int length;
+	void (*print)(struct field_data *fd);
+};
+
+struct display_field_data_info {
+	int length;
 	void (*print)(struct field_data *fd);
+};
+
+struct display_field_variable {
```

Comment:
this struct is based on display_field struct.
it's added not to modify original display_field struct.